packages/api-clients/open-api/bffApi.yml

```diff
@@ -19878,6 +19878,21 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  "/api-docs":
+    get:
+      security: []
+      tags:
+        - docs
```

Comment:
this tag should maybe be declared in the `tags` section (from line 18), though I think it's not strictly required for all tags to be declared there

on the other hand: do we actually want to track this endpoint in the specification? if it's meant just for development usage, I'm not sure it's useful to have here